python/src/aiconfig/editor/server/server.py

```diff
@@ -57,7 +58,7 @@
 CORS(app, resources={r"/api/*": {"origins": "*"}})
 
 
-def run_backend_server(edit_config: EditServerConfig) -> Result[str, str]:
+def run_backend_server(edit_config: EditServerConfig, cli_config_path: str) -> Result[str, str]:
```

Comment:
Can we call this `editor_config_path`. It will contain settings that aren't only specific to CLI

Reply:
sure